public/app/features/explore/TraceView/TraceView.tsx

```diff
@@ -39,6 +39,7 @@ function noop(): {} {
 }
 
 type Props = {
+  datasource: DataSourceApi;
```

Comment:
Where's the datasource getting used? Might be missing something but I don't think we need this